.github/workflows/formula-update.yml

```diff
@@ -32,14 +32,15 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Update Homebrew Formula for ${{ env.PROJECT }}
-        run: sed -i 's/:tag => "[^"]*"/:tag => "${{  env.RELEASE_TAG }}"/' ./Formula/${{ env.PROJECT }}.rb
+        run: |
+          sed -i 's/tag: "[^"]*"/tag: "${{  env.RELEASE_TAG }}"/' ./Formula/${{ env.PROJECT }}.rb
 
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v5.0.2
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
nit: I believe you can drop the extra token if you add:
```yml
permissions:
  contents: write
  pull-requests: write
```

see: https://github.com/SiaFoundation/benchmark/blob/master/.github/workflows/bench.yml